src/clusterfuzz/_internal/build_management/build_manager.py

```diff
@@ -872,6 +897,18 @@ def _unpack_custom_build(self):
                                      build_local_archive):
       return False
 
+    build_download_time = time.time() - download_start_time
+    monitoring_metrics.JOB_BUILD_RETRIEVAL_TIME.add(
+        build_download_time,
+        {
+            # The concept of a fuzz target does not apply
+            # to blackbox fuzzers
```

Comment:
nit: it's more that custom builds don't support fuzz targets today, so a better comment would be:

"Custom builds don't support fuzz targets."

(We could probably fix this if needed -- I think we did support this in the past right @jonathanmetzman ?)